packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -221,15 +222,16 @@ export const Dialog: ComponentWithAs<'div', DialogProps> &
   });
 
   React.useEffect(() => {
+    const target = contentRef?.current;
     if (open) {
-      lockBodyScroll(context.target);
+      disableBodyScroll(target);
     }
     return () => {
       if (open) {
-        unlockBodyScroll(context.target);
+        clearAllBodyScrollLocks(target);
```

Comment:
```suggestion
        enableBodyScroll(target);
```

This should be `enableBodyScroll()`, according to docs it will clean all locks:

> // 4. Useful if we have called disableBodyScroll for multiple target elements,
> // and we just want a kill-switch to undo all that.
> bodyScrollLock.clearAllBodyScrollLocks();